sdk/filter/README.md

```diff
@@ -0,0 +1,24 @@
+# Go Agent Filter
+Filtering capability as specified in [Hypertrace filter specification](https://github.com/hypertrace/specification/blob/main/agent/filtering.md) can be added to instrumentations.
+
+Example
+```go
+// Filter requests containing "foo" in URL
+type FooURLFilter struct {
+}
+
+func (FooURLFilter) EvaluateURL(span sdk.Span, url string) bool {
+	if strings.Contains(url, "foo") {
```

Comment:
I think it is easier to just return `strings.Contains(url, "foo")`

Reply:
right, done.